--- kimageformats/CMakeLists.txt	2023-08-05 16:14:58
+++ CMakeLists.txt	2023-08-05 16:14:49
@@ -74,11 +74,15 @@
 add_feature_info(LibJXL LibJXL_FOUND "required for the QImage plugin for JPEG XL images")
 
 # note: module FindLibRaw missing from https://invent.kde.org/frameworks/extra-cmake-modules
-find_package(LibRaw 0.20.2)
-set_package_properties(LibRaw PROPERTIES
+find_package(libraw CONFIG)
+get_property(importTargetsAfter DIRECTORY "${CMAKE_SOURCE_DIR}" PROPERTY IMPORTED_TARGETS)
+set_package_properties(libraw PROPERTIES
     TYPE OPTIONAL
     PURPOSE "Required for the QImage plugin for RAW images"
 )
+# Adapt naming so the rest of the cmake infra finds this new target
+add_library(LibRaw::LibRaw ALIAS libraw::raw)
+set(LibRaw_FOUND ${libraw_FOUND})
 
 ecm_set_disabled_deprecation_versions(
     QT 5.15.2
