--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -75,11 +75,13 @@
 endif()
 add_feature_info(LibJXL LibJXL_FOUND "required for the QImage plugin for JPEG XL images")
 
-find_package(LibRaw 0.20.2)
+find_package(LibRaw 0.20.2 NAMES libraw)
 set_package_properties(LibRaw PROPERTIES
     TYPE OPTIONAL
     PURPOSE "Required for the QImage plugin for RAW images"
 )
+# Adapt naming so the rest of the cmake infra finds this new target
+add_library(LibRaw::LibRaw ALIAS libraw::raw)
 
 option(KIMAGEFORMATS_JXR "Enable plugin for JPEG XR format" OFF)
 if(KIMAGEFORMATS_JXR)
